src/lib/stack/stack.ts

```diff
@@ -0,0 +1,94 @@
+import { CustomElement, attachShadowTemplate, ICustomElement, coerceBoolean, FoundationProperty, coerceNumber } from '@tylertech/forge-core';
+import { StackAdapter } from './stack-adapter';
+import { StackFoundation } from './stack-foundation';
+import { STACK_CONSTANTS, StackAlignMode } from './stack-constants';
+
+import template from './stack.html';
+import styles from './stack.scss';
+import { BaseComponent } from '../core';
+
+export interface IStackComponent extends ICustomElement {
+  inline: boolean;
+  wrap: boolean;
+  stretch: boolean;
+  gap: number;
+  align: StackAlignMode;
```

Comment:
We may want to rename this to `alignment` since HTML already has an `align` attribute and it automatically applies `text-align` when this attribute is set.